snc.sh

```diff
@@ -132,11 +135,11 @@ ${YQ} eval-all --inplace 'select(fileIndex == 0) * select(filename == "cvo-overr
 ${YQ} eval --inplace ".spec.domain = \"apps-${CRC_VM_NAME}.${BASE_DOMAIN}\"" ${INSTALL_DIR}/manifests/cluster-ingress-02-config.yml
 # Add master memory to 12 GB and 6 cpus 
```

Comment:
Can you update this comment as well?

Reply:
Or even better, add a `echo` printing the actual amount of memory and CPU which will be used as now they are no longer hardcoded.

Reply:
I updated the comments and added echo for each variable: CRC_VM_MEMORY, CRC_VM_CPUS and CRC_VM_DISK_SIZE